.gitignore

```diff
@@ -1,12 +1,12 @@
 # Redis dump files
 dump.rdb
 
-# Byte-compiled / optimized / DLL files
+# Byte-compiled / optimized / DLL files.github/workflows/gsy-e_ci.yaml
 __pycache__/
 *.py[cod]
 *$py.class
 
-# JetBeans PyCharm files
+# JetBeans PyCharm files.github/workflows/gsy-e_ci.yaml
```

Comment:
```suggestion
# JetBeans PyCharm files
```

Reply:
Thanks, got added due to quick global searches